deps/bzip2.BUILD.bazel

```diff
@@ -0,0 +1,166 @@
+"""Agent specific BUILD file for bzip2."""
+
+load("@rules_cc//cc:cc_binary.bzl", "cc_binary")
+load("@rules_cc//cc:cc_library.bzl", "cc_library")
+load("@rules_cc//cc:cc_shared_library.bzl", "cc_shared_library")
+load("@rules_license//rules:license.bzl", "license")
+load("@rules_pkg//pkg:install.bzl", "pkg_install")
+load("@rules_pkg//pkg:mappings.bzl", "pkg_files", "pkg_mklink")
+
+package(
+    default_applicable_licenses = [":license"],
+    default_visibility = ["//visibility:private"],
+)
+
+license(
+    name = "license",
+    # This is the current SPDX known version of the bzip2 license
+    license_kinds = ["@rules_license//licenses/spdx:bzip2-1.0.6"],
+    license_text = "LICENSE",
+    visibility = ["//visibility:public"],
+)
+
+exports_files([
+    "LICENSE",
+])
+
+# only for test cases in //deps/bzip
+exports_files([
+    "sample1.ref",
+    "sample2.ref",
+    "sample3.ref",
+    "sample1.bz2",
+    "sample2.bz2",
+    "sample3.bz2",
+])
+
+_PUBLIC_HEADERS = [
+    "bzlib.h",
+]
+
+cc_binary(
+    name = "bzip2",
+    srcs = ["bzip2.c"],
+    deps = [
+        ":bz2",
+    ],
+    copts = [
+        "-I.",
+        "-Wall",
+        # This is probably obsolete with modern compilers, but that is what
+        # the upstream project uses.
+        "-Winline",
+        "-D_FILE_OFFSET_BITS=64",
+    ],
+    visibility = ["//visibility:public"],
+)
+
+cc_library(
+    name = "bz2",
+    srcs = [
+        "blocksort.c",
+        "bzlib.c",
+        "bzlib_private.h",
+        "compress.c",
+        "crctable.c",
+        "decompress.c",
+        "huffman.c",
+        "randtable.c",
+    ],
+    hdrs = _PUBLIC_HEADERS,
+    copts = [
```

Comment:
same as above

Reply:
Right again.  I transcribed the 'includes = ["."] from the BCR entry, to the safer form -I., but did not check that it was broken in the first place.

And yet another reason to not trust the BCR for anything but rule implementations.